File: javascript/pauloruan/02-football-points/footballPoints.js
function footballPoints(wins, draws, losses) {
	winsPoints = 3 * wins 
	drawsPoints = 1 * draws
	lossesPoints = 0 * losses
	total = winsPoints + drawsPoints + lossesPoints
  return total
}

console.log(footballPoints(3, 4, 2))

File: java/leorv/FootballPoints.java
import java.util.Scanner;

public class FootballPoints {
    public static void main(String[] args) {
        if (args.length != 3) {
            System.out.println("Número de argumentos inválidos.");
        } else {
            System.out.println(footballPoints(
                    Integer.parseInt(args[0]),
                    Integer.parseInt(args[1]),
                    Integer.parseInt(args[2])));
        }
    }

    static int footballPoints(int wins, int draws, int losses) {
        return wins * 3 + draws * 1 + losses * 0;
    }
}